release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -39,9 +39,17 @@ that pertain to {product-title} 3.11 are included in this topic.
 the latest packages from Extras, including CRI-O 1.11 and Docker 1.13. It is
 also supported on Atomic Host 7.5 and later.
 
-{product-title} 3.11 is supported on Red Hat Enterprise Linux 7 nodes running in
+{product-title} 3.11 is supported on Red Hat Enterprise Linux (RHEL) 7 nodes running in
 Federal Information Processing Standards (FIPS) mode.
 
+[NOTE]
```

Comment:
@knewcomerRH, is this final FIPS note for 3.11 acceptable?

Reply:
I think we want to say something more like this: "If you need the East / West traffic on your cluster to be encrypted with FIPS validated crypto, you need to use IPSec." I would avoid calling out master node and worker node. @knobunc for review and revision. Ben, this is for 3.11.

Reply:
@knobunc, @knewcomerRH, will you PTAL at this updated text?

Reply:
@kalexand-rh Text you sent me in chat looks good "If you need the network traffic between nodes, also known as east-west encryption, to be encrypted with FIPS validated cryptography,
xref:../admin_guide/ipsec.adoc#admin-guide-ipsec[encrypt traffic between nodes with IPsec]."